src/less/tab/ds6/tab.less

```diff
@@ -10,14 +10,30 @@
     border-bottom: 1px solid @ds6-color-white; // prevent margin-collapse on next element
 }
 
+// move the thin grey bar up by 1 pixel
+.tabs__content,
+.fake-tabs__content {
+    border-top: 1px solid #c7c7c7;
+    position: relative;
+    top: -1px;
+}
+
+// put the tab, with it's blue bar, onto a layer on top of the thin grey bar
+div.tabs__items[role="tablist"],
+ul.fake-tabs__items {
+    position: relative;
+    z-index: 1;
+}
+
 // outer
 div.tabs__item[role="tab"],
 li.fake-tabs__item {
+    border-bottom: 3px solid transparent;
```

Comment:
In the fake-tabs case, why is the border on the `<li>` and not on the anchor? It makes for a weird focus ring which doesn't match the other tabs.

**Regular tab**

![image](https://user-images.githubusercontent.com/105656/53784974-25391e80-3ed4-11e9-9ddc-e4d547bfe457.png)

**Fake tab**

![image](https://user-images.githubusercontent.com/105656/53784986-371ac180-3ed4-11e9-999a-27e683b04323.png)

Is that just to simplify the CSS selectors? Or another reason I'm not understanding?

Reply:
Hmm, not sure. I believe it was already like that and I just moved it out of base to ds6.

Reply:
Changed it to anchor and it breaks the position of the gray bar in Safari.

Before change:

<img width="1017" alt="screen shot 2019-03-05 at 9 12 56 am" src="https://user-images.githubusercontent.com/38065/53823698-2182b780-3f27-11e9-834d-ff1806ecab32.png">


After change:

<img width="1025" alt="screen shot 2019-03-05 at 9 12 40 am" src="https://user-images.githubusercontent.com/38065/53823707-247da800-3f27-11e9-9f29-3a962af087c1.png">

I'll maybe just leave it as it was for now and I can open a separate ticket to go on the backlog.

Reply:
Yeah, let's create a task. I had played with it and we will probably need some margin changes, etc., to accomplish it.